docs_src/getting-started/quick-start/index.md

```diff
@@ -1,227 +1,255 @@
 ```
 docker-compose up -d
 ```
-Finally, verify that the EdgeX containers have all been started:
+The -d option on docker-compose up is to start the containers in detached mode which allows the containers to run independently from the terminal window.  Finally, verify that the EdgeX containers have started:
 ```
 docker-compose ps 
 ```
-!!! Note 
-    Initialization or seed containers, like config-seed, will have exited as their job is just to initialize the associated service and then exit.
-
 ![image](EdgeX_GettingStartedUsrActiveContainers.png)
-
-The following table captured the Default Service Ports. This table provides the default ports used by each of the EdgeX micro services (per its default configuration and the EdgeX provided docker-compose files).  These default ports are also used in the EdgeX provided service routes defined in the Kong API Gateway for access control.
-
-
-|Services Name|	Port Definition|
-|---|---|
-|consul	|8400|
-| 	|8500|
-| 	|8600|
-|vault	|8200|
-|kong-db	|5432|
-|kong	|8000|
-| 	|8001|
-| |	8443|
-| 	|8444|
-|mongo|	27017|
-|redis|	6379|
-|system	|48090|
-|core-data|	48080|
-| 	|5563|
-|core-metadata	|48001|
-|core-command	|48082|
-|support-notifications	|48060|
-|support-logging	|48061|
-|support-scheduler|	48085|
-|device-virtual	|49990|
-|device-random	|49988|
-|device-mqtt	|49982|
-|device-modbus	|49991|
-|device-snmp	|49993|
-|UI	|4000|
-
+*If all EdgeX containers pulled and started correctly and without error, you should see a process status (ps) that looks similar to the image above.*
 
 ## Connecting a Device
-
-
-EdgeX Foundry provides a [Random Number Device Service](https://github.com/edgexfoundry/device-random) which is useful to testing, it returns a random number within a configurable range. Configuration for running this service was included in the `docker-compose.yml` file you downloaded at the start of this guide, but it is disabled by default. To enable it, uncomment the following lines in your `docker-compose.yml`:
+EdgeX Foundry provides a [Random Number device service](https://github.com/edgexfoundry/device-random) which is useful to testing, it returns a random number within a configurable range. Configuration for running this service is in the `docker-compose.yml` file you downloaded at the start of this guide, but it is disabled by default. To enable it, uncomment the following lines in your `docker-compose.yml`:
 ``` yaml
-device-random:
-    image: edgexfoundry/docker-device-random-go:1.1.0
+  device-random:
+    image: edgexfoundry/docker-device-random-go:1.2.0
     ports:
-    - "49988:49988"
+      - "127.0.0.1:49988:49988"
     container_name: edgex-device-random
     hostname: edgex-device-random
     networks:
-    - edgex-network
-    volumes:
-    - db-data:/data/db
-    - log-data:/edgex/logs
-    - consul-config:/consul/config
-    - consul-data:/consul/data
+      edgex-network:
+        aliases:
```

Comment:
i dont think we need the alias here as container name and hostname are specified above

Reply:
fixed